torch/nn/utils/parametrizations.py

```diff
@@ -104,10 +102,17 @@ def forward(self, weight: torch.Tensor) -> torch.Tensor:
             weight_mat = self._reshape_weight_to_matrix(weight)
             if self.training:
                 self._power_method(weight_mat, self.n_power_iterations)
+            # See above on why we need to clone
+            if self.n_power_iterations > 0:
```

Comment:
`n_power_iterations` is always positive, so this if is unnecessary.
https://github.com/pytorch/pytorch/blob/a3eec40b557db06220af8049fd605e19be39f3ee/torch/nn/utils/parametrizations.py#L22-L24